docs/readthedocs/models/input/grid/gridcontainer.md

```diff
@@ -37,17 +37,49 @@ A synoptic overview of both classes' attributes is given here:
 
 ## Attributes, Units and Remarks
 
-| Attribute               | Unit | Remarks                                           |
-|:------------------------|:-----|:--------------------------------------------------|
-| gridName                | --   | Human readable identifier                         |
-| rawGrid                 | --   | see below                                         |
-| systemParticipants      | --   | see below                                         |
-| graphics                | --   | see below                                         |
-| subGridTopologyGraph    | --   | topology of sub grids - only `JointGridContainer` |
-| predominantVoltageLevel | --   | main voltage level - only `SubGridContainer`      |
-| subnet                  | --   | sub grid number - only `SubGridContainer`         |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
 
 
+   * - Attribute
+     - Un it
```

Comment:
This should be fixed.

Reply:
I hope I have eliminated all attention errors.